keyboards/the_royal/schwann/schwann.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT_mit( \
+    k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012, \
+    k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112, \
+    k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212, \
+         k32, k33, k34, k35,    k37,   k38, k39, k310, k311        \
+) \
+{ \
+    {k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012}, \
+    {k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112}, \
+    {k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212}, \
+    {XXX, k32, k33, k34, k35, XXX, k37, k38, k39, k310, k311, XXX } \
+}
+
+#define LAYOUT_full_ortho( \
+    k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012, \
+    k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112, \
+    k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212, \
+         k32, k33, k34, k35, k36, k37, k38, k39, k310, k311        \
+) \
+{ \
+    {k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012}, \
+    {k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112}, \
+    {k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212}, \
+    {XXX, k32, k33, k34, k35, k36, k37, k38, k39, k310, k311, XXX } \
+}
+
+#define LAYOUT_big_boy( \
+    k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012, \
+    k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112, \
+    k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212, \
+           k32,                 k37,                 k311        \
+) \
+{ \
+    {k01, k02, k03, k04, k05, k06, k07, k08, k09, k010, k011, k012}, \
+    {k11, k12, k13, k14, k15, k16, k17, k18, k19, k110, k111, k112}, \
+    {k21, k22, k23, k24, k25, k26, k27, k28, k29, k210, k211, k212}, \
+    {XXX, k32, XXX, XXX, XXX,    k37,   XXX, XXX, XXX,  k311, XXX } \
+}
```

Comment:
```suggestion
#define LAYOUT_mit( \
  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, \
  k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, \
  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, \
       k31, k32, k33, k34,    k36,   k37, k38, k39, k3A       \
) \
{ \
  {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B}, \
  {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B}, \
  {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B}, \
  {XXX, k31, k32, k33, k34, XXX, k36, k37, k38, k39, k3A, XXX}  \
}

#define LAYOUT_full_ortho( \
  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, \
  k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, \
  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, \
       k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A       \
) \
{ \
  {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B}, \
  {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B}, \
  {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B}, \
  {XXX, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3A, XXX}  \
}

#define LAYOUT_big_boy( \
  k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B, \
  k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B, \
  k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B, \
         k31,                 k36,                 k3A        \
) \
{ \
  {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k0B}, \
  {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1A, k1B}, \
  {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2A, k2B}, \
  {XXX, k31, XXX, XXX, XXX, XXX, k36, XXX, XXX, XXX, k3A, XXX}  \
}
```
Small adjustment made for a few reasons.  You were indexing from 0 on the rows, but 1 from the cols.  And my personal opinion is stuff like `k111` is ugly because it's not clear at a glance if we're talking about 1 row, column 11 or row 11, column 1.  Indexing both from 0 and using hex numbering makes it a little smoother (and saves a few bytes).

Also, the big_boy layout's electrical matrix was missing a `XXX` -- tossed that in for good measure.